src/components/checks/[share_token]/practice/NavigationButtons.tsx

```
@@ -1,17 +1,16 @@
 
-import { ButtonHTMLAttributes } from 'react'
 import { usePracticeStore } from '@/src/components/checks/[share_token]/practice/PracticeStoreProvider'
-import { Button } from '@/src/components/shadcn/button'
+import { Button, SimpleButtonProps } from '@/src/components/shadcn/button'
 import { cn } from '@/src/lib/Shared/utils'
 
-export function PracticeNavigationNextButton({ ...props }: {} & ButtonHTMLAttributes<HTMLButtonElement>) {
+export function PracticeNavigationNextButton({ ...props }: {} & SimpleButtonProps) {
   const { nextQuestion } = usePracticeStore((store) => store)
 
   return <Button {...props} onClick={() => nextQuestion(true)} children='Next' className={cn('', props.className)} />
```

Biome (2.3.14)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)